types/codec.go

```diff
@@ -13,5 +13,5 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 
 // RegisterInterfaces registers the sdk message type.
 func RegisterInterfaces(registry types.InterfaceRegistry) {
-	registry.RegisterInterface("cosmos.v1beta1.Msg", (*Msg)(nil))
+	registry.RegisterInterface("cosmos.base.v1beta1.Msg", (*Msg)(nil))
```

Comment:
btw, IBC called their base `core`, anyone has some strong opinion if we want to do the same for `cosmos.core.*` everywhere?